lib/mayaUsd/fileio/translators/translatorMayaReference.cpp

```diff
@@ -414,7 +417,13 @@ MStatus UsdMayaTranslatorMayaReference::update(const UsdPrim& prim, MObject pare
     // If no reference found, we'll need to create it. This may be the first time we are
     // bring in the reference or it may have been imported or removed directly in maya.
     if (refNode.isNull()) {
-        return LoadMayaReference(prim, parent, mayaReferencePath, rigNamespaceM);
+        bool mergeNamespacesOnClash = false;
+        if (UsdAttribute mergeNamespacesOnClashAttribute = prim.GetAttribute(m_mergeNamespacesOnClash))
```

Comment:
I'm happy that we are not promoting it yet to the actual maya reference schema attribute. Let's put it in use and see later if we should.